src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2085,6 +2087,8 @@ because xbuild doesn't support framework reference assemblies.
       ApplicationSharedLibraries="@(_ApplicationSharedLibrary)"
       DebugBuild="$(AndroidIncludeDebugSymbols)"
       AndroidBinUtilsDirectory="$(AndroidBinUtilsDirectory)"
+      EnableMarshalMethodTracing="$(_AndroidEnableMarshalMethodTracing)"
```

Comment:
What happens when `$(_AndroidEnableMarshalMethodTracing)`=True and `$(_AndroidNdkDirectory)` isn't set and/or is the "internal" clang tooling?  What's the error message look like?  Is that error message acceptable?

(Sure, this is a *private* property *for now*, but even we make mistakes…)

Reply:
For now it's `ArgumentNullException(nameof (ndk))` in the method where we actually [use the NDK](https://github.com/xamarin/xamarin-android/pull/7927/files#diff-ecb7cc6ca53340e73cbe5cd352b851970b5cc5756a15c68dc94647b0925dc9c3R238). Not sure if we ever want to make this property "public", though.

Reply:
No longer a problem, NDK not required anymore